spec/features/blog_spec.rb

```diff
@@ -15,11 +15,13 @@
     expect(page).to have_content("Blog posts about #{tag}")
   end
 
-  shared_examples "paginating blog posts" do |path|
+  shared_examples "paginating blog posts" do |path, count, next_page|
```

Comment:
@plisovin Just want to check what these changes do and why we need them here? Thanks!